WooCommerce/Classes/Analytics/WooAnalyticsEvent+TestOrder.swift

```diff
@@ -0,0 +1,39 @@
+import Foundation
+
+extension WooAnalyticsEvent {
+    enum TestOrder {
+        enum Keys {
+            static let isWooExpressStore = "is_wooexpress_store"
```

Comment:
❓  This sounds like a useful value. Would it make sense to add this property to all events? Just like `is_wpcom_store` and `is_wpcom_store`

Reply:
Good idea! I'll look into that in a separate PR 🤔

Reply:
Good idea! Asking in the team channel to see if we should do this [p1690456615750299-slack-C03L1NF1EA3]. If we do, I'll handle the change in a separate PR.